lib/bundlex/project.ex

```diff
@@ -11,6 +11,8 @@ defmodule Bundlex.Project do
 
   @src_dir_name "c_src"
 
+  @dep_format_interfaces %{nifs: :nif, cnodes: :cnode, ports: :port}
```

Comment:
dep is associated rather with dependency than deprecated

Reply:
fixed